### PR TITLE
drakrun: attach snapshot modification time to the analysis reports

### LINKED ISSUE #156: Basic information about snapshot in the analysis artifacts
There should be some analysis artifact which would allow to distinguish between different VM snapshots. It could be some snapshot hash or sth.

### PR DESCRIPTION
**Pull request by icedevml**
resolve #156

### CODE REVIEW

File: drakrun/drakrun/run.py [hunk 1]
```diff
@@ -35,3 +36,5 @@ def run_vm(vm_id):
 
         with open("/var/log/xen/qemu-dm-vm-{vm_id}.log".format(vm_id=vm_id), "rb") as f:
             logging.error(f.read())
+
+    return snapshot_version
```
Comment (icedevml) [inline drakrun/drakrun/run.py:40]: idk if this is sane idea to return it from `run_vm` but at least it was simple :thinking:

**Comment by chivay** [comment]
I have a general feeling that `run.py` as a separate file isn't really necessary.
Maybe it's a good time to move this function into `main.py`?

**Comment by icedevml** [comment]
> I have a general feeling that `run.py` as a separate file isn't really necessary.
> Maybe it's a good time to move this function into `main.py`?

done

**Review by chivay — APPROVED** [review]

**Pull request merged**